java/src/test/java/com/google/oak/crypto/ClientEncryptorTest.java

```diff
@@ -17,9 +17,14 @@
 package com.google.oak.crypto;
 
 import com.google.oak.crypto.ClientEncryptor;
+import com.google.oak.crypto.Encryptor;
 import com.google.oak.util.Result;
+import com.google.protobuf.ByteString;
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
+import oak.crypto.AeadEncryptedMessage;
```

Comment:
Will update in a separate proto, since our remote attestation messages also don't have `com.google.` in them: https://github.com/project-oak/oak/blob/096046a6b1fd3a502a74cc3522aec97958f7d1ee/oak_remote_attestation_noninteractive/proto/v1/messages.proto#L19-L22

Reply:
https://github.com/project-oak/oak/issues/3882